### PR TITLE
Admin disable user #2

### PR DESCRIPTION
**Pull request by n-flint**
Created a test file for admin changing the role/status of a user

Added a enable and disable method to the admin/users controller

Created custom routes to route to the enable or disable action

changed my email in the readme

### CODE REVIEW

**Comment by Vjp888** [comment]
Resolved the merge conflict, otherwise the code looks good and well tested.

I approve for merge.

**Comment by renecasco** [comment]
Good to merge,

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Little Shop is a web based application which runs on the Ruby On Rails framework
 - Rene Casco – rene.casco@icloud.com
 - Ty Mazey - 43197720+TyMazey@users.noreply.github.com
 - Vincent Provenzano - 40582072+Vjp888@users.noreply.github.com
-- Noah Flint - 421236+n-flint@users.noreply.github.com
+- Noah Flint - 34421236+n-flint@users.noreply.github.com
```
Comment (TyMazey) [inline README.md:16]: Nice! glad to have you apart of the team!

**Review by TyMazey — APPROVED** [review]
looks good noah i approve

**Pull request merged**